qiita_pet/handlers/api_proxy/sample_template.py

```diff
@@ -23,6 +23,33 @@
 from qiita_pet.handlers.api_proxy.util import check_access, check_fp
 
 
+def sample_template_get_req(samp_id, user_id):
+    """Gets the json of the full sample template
+
+    Parameters
+    ----------
+    samp_id : int
+        SampleTemplate id to get info for
+    user_id : str
+        User requesting the sample template info
+
+    Returns
+    -------
+    dict of dictionaries
+        Dictionary object where the keys are the metadata samples
+        and the values are a dictionary of column and value.
+        Format {sample: {column: value, ...}, ...}
+    """
+    template = SampleTemplate(int(samp_id))
```

Comment:
There have been a few times in past code where the direct input from a form was passed, so a string was passed instead of an int. The object still works in most cases with the string, so it's really hard to debug the few cases where it breaks. I put the explicit cast just to make absolutely sure it is an int when it goes into the instantiation.